src/models/OtrStore.js

```diff
@@ -53,6 +56,34 @@ OtrStore.prototype.initChan = function(args) {
 			session.on('io', function(msg, meta) {
 				irc.send(chan.name, msg);
 			});
+
+			session.on('error', function(err, severity)  {
+				// severity is one of "warning" or "error"
+				var body = {type: severity, msg: err};
```

Comment:
This comment still applies but since we need that PR I'll close my eyes. We'll probably do a cleanup session when everything gets done anyway.